lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -180,9 +180,10 @@ const Reportback = {
    * Get Kudos Reactions for a Reportback item.
    *
    * @param  {array} data  Array of Kudos objects from API.
+   * @param  {int} kudosId if from Rogue.
    * @return {array}
    */
-  getReactions: function(data) {
+  getReactions: function(data, kudosId = null) {
```

Comment:
@DFurnes had to update this a bit for the same logic we worked on together this am! When a user has already liked a post, the front end needs to know about the `kudosId` so passing it here. This fixes the logic to make sure if they `unlike` it, it makes a `DELETE` request instead of a `POST` now knowing it's `kudosId`